Source/WebCore/dom/Node.h

```diff
@@ -488,7 +488,7 @@ class Node : public EventTarget {
 
 #if ENABLE(TREE_DEBUGGING)
     void showNode(const char* prefix = "") const;
-    void showTreeForThis() const;
+    WEBCORE_EXPORT void showTreeForThis() const;
```

Comment:
This change should be undone.

Reply:
why exactly would it be bad to leave it in? I find myself often adding it for debugging, which results in a mostly full-rebuil because of the header it's in

Reply:
I think that's a good reason to keep this! Can you mention that in the ChangeLog?